### PR TITLE
Restrict access on SG of masters

### PR DESCRIPTION
**Pull request by renatoc**
Restrict access to masters using SG of elb

### CODE REVIEW

File: modules/aws/cnid-vpc/sg-elb.tf [hunk 1]
```diff
@@ -17,7 +17,7 @@ resource "aws_security_group" "api" {
 
   ingress {
     protocol    = "tcp"
-    cidr_blocks = ["0.0.0.0/0"]
+    cidr_blocks = "${var.master_sg_ingress_cidr}"
```
[the remark this answers is not in the document]
Comment (renatoc) [reply modules/aws/cnid-vpc/sg-elb.tf]: https://github.com/conde-nast-international/cnid-infrastructure-core-platform/blob/master/dev/eu-central-1/kubernetes/override.tf#L116

**Comment by tomgco** [comment]
This change would make the Kubernetes API be unreachable from our CI systems, maybe update the override first?

**Comment by renatoc** [comment]
> This change would make the Kubernetes API be unreachable from our CI systems, maybe update the override first? 

It will not, because after merging we will create PR's on config's to have CI and that's when it's applied

**Comment by katiegamanji** [comment]
How was this tested? Are we certain that no connection is cut of?

**Comment by renatoc** [comment]
> How was this tested? Are we certain that no connection is cut of?

I have terraform plan for dev. But there only way to test this is merge and after change config for dev and apply

**Pull request merged**